schema/swagger.yml

```diff
@@ -595,6 +595,46 @@ paths:
           schema:
             $ref: '#/definitions/UnexpectedError'
 
+  /delegates/{publicKey}/forged:
```

Comment:
We already using some endpoints which have **address** in the API path. 

```
/accounts/{address}/multisignature_groups
/accounts/{address}/multisignature_memberships
```

Delegate is also an account. It does not make logic that we identify an account with address and delegate with publicKey. We should stay consistent. The decision to use `address` as path param was done after a bit discussion with @karmacoma , so we should stick to that. And expose that endpoint as;

```
/delegates/{address}/stats:
```

I understand it requires some change in `modules.blocks.utils.aggregateBlocksReward` but that's not that much and worth doing. Just doing a call to `logic.account.get({address: ''}, ['publicKey'])` will give you public key to be used for rest of implementation of `modules.blocks.utils.aggregateBlocksReward`